packages/react/src/Header/Header.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { Header, HeaderProps } from './Header'
+import '@testing-library/jest-dom'
+
+describe('Header', () => {
+  const defaultProps: HeaderProps = {
+    logoLink: '/',
+  }
+
+  it('renders', () => {
+    const { container } = render(<Header {...defaultProps} />)
+
+    const component = container.querySelector(':only-child')
```

Comment:
[Testing library recommends getting elements by role if that's possible](https://testing-library.com/docs/queries/about/#priority). So this should be something like `screen.getByRole('banner')`. Same for the link and heading

Reply:
Ah right, didn't change it from the plop template.